From c542c259ba9e71a5ade235dfbcde67ac8b510c4f Mon Sep 17 00:00:00 2001
From: Sebastien GODARD <sysstat@users.noreply.github.com>
Date: Fri, 19 Apr 2019 15:20:45 +0200
Subject: [PATCH] sar: Add missing %gnice CPU value for tickless CPU

A field (which should be displayed as 0.00) was missing in CPU
statistics displayed by "sar -u ALL" for tickless CPUs.

Signed-off-by: Sebastien GODARD <sysstat@users.noreply.github.com>
---
 pr_stats.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pr_stats.c b/pr_stats.c
index 4fbd5d1..319630d 100644
--- a/pr_stats.c
+++ b/pr_stats.c
@@ -218,8 +218,8 @@ __print_funct_t print_cpu_stats(struct activity *a, int prev, int curr,
 				 * %irq, %soft, %guest, %gnice.
 				 */
 				else if (DISPLAY_CPU_ALL(a->opt_flags)) {
-					cprintf_pc(DISPLAY_UNIT(flags), 4, 9, 2,
-						   0.0, 0.0, 0.0, 100.0);
+					cprintf_pc(DISPLAY_UNIT(flags), 5, 9, 2,
+						   0.0, 0.0, 0.0, 0.0, 100.0);
 					printf("\n");
 				}
 				continue;
-- 
2.30.1 (Apple Git-130)

